pumpkin/src/command/commands/cmd_pumpkin_test_client_side_arg_parsers.rs

```diff
@@ -0,0 +1,112 @@
+static TEAM: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Team));
+static ITEMSLOT: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::ItemSlot));
+static CONTAINERSLOT: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::ContainerSlot));
+static RESOURCELOCATION: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::ResourceLocation));
+static FUNCTION: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Function));
+static ENTITYANCHOR: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::EntityAnchor));
+static INTRANGE: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::IntRange));
+static FLOATRANGE: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::FloatRange));
+static DIMENSION: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Dimension));
+static GAMEMODE: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Gamemode));
+static TIME: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Time { min: 0 }));
+static RESOURCEORTAG: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::ResourceOrTag { identifier: "item" }));
+static RESOURCEORTAGKEY: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::ResourceOrTagKey { identifier: "item" }));
+static RESOURCE: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Resource { identifier: "item" }));
+static RESOURCEKEY: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::ResourceKey { identifier: "item" }));
+static TEMPLATEMIRROR: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::TemplateMirror));
+static TEMPLATEROTATION: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::TemplateRotation));
+static HEIGHTMAP: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Heightmap));
+static UUID: LazyLock<ClientSideArgParserTester<'static>> = LazyLock::new(|| ClientSideArgParserTester(ProtoCmdArgParser::Uuid));
+
+pub(super) fn pumpkin_test_client_side_arg_parsers() -> NonLeafNodeBuilder<'static> {
```

Comment:
Also like said these Types can change. So we would have to also change the code all the time